### PR TITLE
rgw-admin: support for processing all gc objects including unexpired.

### PR DESCRIPTION
**Pull request by yaozongyou**
Signed-off-by: Yao Zongyou <yaozongyou@vip.qq.com>

### CODE REVIEW

File: src/rgw/rgw_admin.cc [hunk 1]
```diff
@@ -156,7 +156,8 @@ void usage()
   cout << "  usage trim                 trim usage (by user, date range)\n";
   cout << "  gc list                    dump expired garbage collection objects (specify\n";
   cout << "                             --include-all to list all entries, including unexpired)\n";
-  cout << "  gc process                 manually process garbage\n";
+  cout << "  gc process                 manually process garbage (specify\n";
+  cout << "                             --include-all to process all entries, including unexpired)\n";
```
Comment (yehudasa) [inline src/rgw/rgw_admin.cc]: @yaozongyou need also to update the template at `src/test/cli/radosgw-admin/help.t`

Comment (yaozongyou) [reply src/rgw/rgw_admin.cc]: the help.t has been already updated, see the last file diffs.

**Comment by yaozongyou** [comment]
rebase it on master

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2017-11-25_00:12:48-rgw-wip-kefu-testing-2017-11-25-0120-distro-basic-mira/

@yehudasa is this ready to merge?

**Comment by yuriw** [comment]
[wip-yuri4-testing](https://trello.com/c/8R4bvb7Y/212-wip-yuri4-testing)

**Pull request merged**